.ci.yaml

```diff
@@ -8,9 +8,40 @@
 enabled_branches:
   - master
 
+platform_properties:
+  windows:
+    properties:
+      caches: >-
+        [
+          {"name": "vsbuild", "path": "vsbuild"},
+          {"name": "pub_cache", "path": ".pub-cache"}
+        ]
+      dependencies: >
+        [
+          {"dependency": "certs"}
+        ]
+      device_type: none
+      os: Windows
+
 targets:
   - name: Windows Plugins
```

Comment:
If this is master, can we name it "Windows Plugins master channel" for clarity?

Reply:
Of course, I sent https://flutter-review.googlesource.com/c/infra/+/16745 as we need to add a try builder before we can rename it here